src/webots/sound/WbSoundClip.hpp

```diff
@@ -42,6 +42,9 @@ class WbSoundClip {
   unsigned int mBuffer;
   int mSide;  // 0: both sides, -1: left only, 1: right only
   double mBalance;
+
+private:
+  bool mAudio;
```

Comment:
I would call it mOpenAL instead, it sounds more precise to me.
```suggestion
  bool mOpenAL;
```